src/firmware_new/boards/arduino_uno/serial_echo_example.cc

```diff
@@ -0,0 +1,22 @@
+/*
+ * Echoes what is sent back through the serial port.
+ *
+ */
+
+
+
+#ifdef PLATFORMIO_BUILD
+#include "echo_library.h"  // This is how PlatformIO sees and includes the library.
+#else
+#include "echo_library.h"  // actual path to file (in this example its the same)
+#endif
```

Comment:
This doesn't seem like it does anything?

Reply:
For convenience and clarity i think its best to include the actual path of the file (for the benefit of developers navigating the codebase), as well as the path that is actually used when PlatformIO compiles (which is just `file_name.h`. 

In this specific case it doesnt matter, but I could move `echo_library.h` to another directory to make the difference noticeable. Or do you think all of it is unnecessary?

Reply:
Ah no thats fine, when we write the e-stop code, we can look at how it looks like again.